.github/workflows/test_template.yml

```
@@ -1,4 +1,4 @@
-name: Test Template
+name: Test Template's codebase
```

`'s` is more for people possessions 🤔 

```suggestion
name: Test Template Codebase
```

of 

```suggestion
name: Test the codebase of the template
```